### ISSUE TITLE
[Weather] [Crop & Soil] Remove Irrigation data from Weather

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#1509]
Irrigation is a field management practice, not a weather condition.

## Urgency Level

- [ ] Urgent: a critical feature or fix is needed
- [ ] High: the issue significantly impacts the user experience
- [ ] Medium: the issue affects functionality but has a workaround
- [x] Low: the issue doesn't have a significant immediate impact
- [ ] Wishful: the issue should not be prioritized over any others
- [ ] Unsure: the urgency level has not been or cannot be assessed at this time 

## Status

Currently irrigation data is a column in every weather input file. It should be separated out and either put in its own file or combined with one of the field data input files.

## Intention

The expected outcome is to separate irrigation data from weather data which will make inputs more modular and will therefore help weather data adhere to single-responsibility principles. If we start using other sources of weather data outside RuFaS input files (APIs, external databases), there is a high likelihood that irrigation data won't be a part of that dataset. Separating these out means RuFaS can become more flexible to using different weather input sources.

## Request

Currently in a weather input CSV file, the structure of the file looks like this:

```
year,jday,precip,high,low,avg,Hday,irrigation
1980,1,0,0.5,-6,-2.75,4.883742794,0
```

We would like to extricate that final column of irrigation data to its own file or to be put into a field data file if that seems appropriate.

Please take into account any activity on issue #1372 which is associated with this issue.

**Comment by Isgotkowitz** [comment]
Blocked by #1507.

**Comment by ew3361zh** [comment]
Unblocking this - #1507 is completed.

**Comment by ew3361zh** [comment]
Conversation with Kristan 9/9/24 in Slack:

Niko:
Question for the SoilCrop folks - I'm migrating irrigation input data out of weather input and it looks like irrigation is only ever a list of 0s (across all weather inputs on the dev branch currently).
Is there a dataset that has populated irrigation data that someone could share with me?
Does irrigation need to be an input?

Kristan Reed:
Hi Niko! The short answers are yes and yes!
If you look on the pilot_testing_Dec2023 branch, there are a few weather datasets that are labeled with irrigation
For inputs, yes, we need a way for the user to indicate when and how much irrigation is used but this would be a good question for our new C&S SMEs and possibly a conversation for the annual meeting to figure out a better way for this information to be provided.


Will be putting this issue on hold until after C&S SME has had an opportunity to weigh in on design.

**Comment by KFosterReed** [comment]
Groomed March 27, 2026: This issue is still relevant however a lower priority to improving the overal irrigation methods. I made this a sub-issue to #1900 .
